geometry_msgs/Point screen_point
geometry_msgs/Point[] data_points
